package.json

```diff
@@ -40,7 +40,7 @@
     "@babel/polyfill": "^7.0.0",
     "chalk": "^2.1.0",
     "figures": "^2.0.0",
-    "ganache-core": "^2.3.2",
+    "ganache-core": "^2.2.1",
```

Comment:
Let's not pin this here, but pin it in aragonCLI so we can release that. Let's keep testing this after the aragonCLI release.

Reply:
@sohkai  I'm not sure about this one. You meant to leave v2.3.2 or do something else?

Reply:
Yes, hope is that everything will work with v2.3.2 (but maybe not D:). If it doesn't, then let's make sure to pin it to 2.2.1.